cs/Serializer/VowpalWabbitSerializerFactory.cs

```diff
@@ -47,25 +42,34 @@ public static class VowpalWabbitSerializerFactory
         /// Compiles a serializers for the given example user type.
         /// </summary>
         /// <typeparam name="TExample">The example user type.</typeparam>
-        /// <param name="visitor">The visitor to be used for serialization.</param>
         /// <param name="settings">The serializer settings.</param>
         /// <returns>A serializer for the given user example type.</returns>
         public static VowpalWabbitSerializer<TExample> CreateSerializer<TExample>(VowpalWabbitSettings settings)
         {
             var serializerFunc = CreateSerializer<TExample, VowpalWabbitInterfaceVisitor, VowpalWabbitExample>();
+
+#if DEBUG
+            var stringSerializerFunc = CreateSerializer<TExample, VowpalWabbitStringVisitor, string>();
+
+            Func<VowpalWabbit, TExample, ILabel, VowpalWabbitExample> wrappedSerializerFunc;
             if (serializerFunc == null)
             {
                 // if no features are found, no serializer is generated
-                serializerFunc = (_,__,___) => null;
+                wrappedSerializerFunc = (_,__,___) => null;
```

Comment:
i found that the _ causes stylecop (or maybe fxcop) to crash, so it might be good to use dummy names instead.